developer/provider/accessdbprovidersample03.md

```diff
@@ -10,45 +10,39 @@ ms.assetid: 9e576199-49c7-4355-9686-f9ed40c64a5f
 caps.latest.revision: 10
 ---
 # AccessDBProviderSample03
-This sample shows how to overwrite the [System.Management.Automation.Provider.Itemcmdletprovider.Getitem*](/dotnet/api/System.Management.Automation.Provider.ItemCmdletProvider.GetItem) and [System.Management.Automation.Provider.Itemcmdletprovider.Setitem*](/dotnet/api/System.Management.Automation.Provider.ItemCmdletProvider.SetItem) methods to support calls to the Get-Item and Set-Item cmdlets. The provider class in this sample derives from the [System.Management.Automation.Provider.Itemcmdletprovider](/dotnet/api/System.Management.Automation.Provider.ItemCmdletProvider) class.
 
-## Demonstrates
-
-> [!IMPORTANT]
```

Comment:
What happened to this content?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.